German Translation of User Guides (Wallet 1/2) (#1332)
monero-project/monero-site: _i18n/de/resources/user-guides/monero-wallet-cli.md

{% include disclaimer.html translated="yes" translationOutdated="no" %}

# monero-wallet-cli

`monero-wallet-cli` ist die im Monero-Baum enthaltene Wallet-Software, die als Konsolenprogramm ein Konto verwaltet. Während ein Bitcoin-Wallet sowohl das Konto als auch die Blockchain verwaltet, ist dies bei Monero getrennt: `monerod` ist für die Blockchain, `monero-wallet-cli` für das Konto zuständig.

Wie verschiedene Vorgänge ausgehend von der Benutzeroberfläche des `monero-wallet-cli` - die Befehlszeile - gesteuert werden, wird in dieser Anleitung erklärt. Es wird angenommen, dass du die neueste Version Moneros nutzt und mithilfe anderer Anleitungen bereits ein Konto erstellt hast.

## Guthaben überprüfen

Da die Zuständigkeiten für Blockchain und Wallet bei verschiedenen Programmen liegen, benötigen einige Dienste des `monero-wallet-cli` eine Verbindung mit einem Hintergrunddienst (darunter die Suche nach eingehenden Transaktionen). 
Sobald du sowohl `monero-wallet-cli` wie auch `monerod` gestartet hast, kannst du `balance` eingeben.

Beispiel: 

> Dies zieht Blöcke aus dem Hintergrunddienst, welche dein Wallet vielleicht noch nicht gesehen hat, und aktualisiert dein Guthaben entsprechend. Dieser Vorgang läuft normalerweise etwa jede Minute im Hintergrund ab. Um dein Guthaben ohne Aktualisierung anzuzeigen:

Reviewer: ```suggestion
Dies lädt Blöcke aus dem Hintergrunddienst, welche dein Wallet vielleicht noch nicht gesehen hat, und aktualisiert dein Guthaben entsprechend. Dieser Vorgang läuft normalerweise etwa jede Minute im Hintergrund ab. Um dein Guthaben ohne Aktualisierung anzuzeigen:
```

### An eine Standardadresse senden

    transfer ADDRESS AMOUNT PAYMENTID

Ersetze `ADDRESS` mit der Adresse, an die du senden möchtest, `AMOUNT` mit dem Betrag der Monero, die du versenden wirst und `PAYMENTID` mit der dir bereitgestellten Zahlungs-ID. Zahlungs-IDs sind optional: Wenn der Empfänger keine benötigt, kannst du sie einfach weglassen.

### An eine integrierte Adresse senden

    transfer ADDRESS AMOUNT

In diesem Fall ist die Zahlungs-ID in die integrierte Adresse inkludiert.

### Die Anzahl an Outputs für eine Transaktion festlegen

    transfer RINGSIZE ADDRESS AMOUNT

Ersetze `RINGSIZE` mit der Anzahl an Outputs, die du verwenden möchtest. **Die Standardeinstellung ist 11, wenn nichts anderes festgelegt wird.** Es ist ratsam, die voreingestellte Ringgröße zu verwenden; du kannst die Zahl aber auch erhöhen, wenn du mehr Outputs beifügen möchtest. Je größer die Zahl ist, desto größer ist auch die Transaktion und die mit ihr einhergehenden Gebühren.

## Monero empfangen

> Wenn du eine eigene Monero-Adresse hast, kannst du dem an dich Sendenden ganz einfach deine Standardadresse geben.

Reviewer: ```suggestion
Wenn du eine eigene Monero-Adresse hast, kannst du jemandem ganz einfach deine Standardadresse geben.
```

Diese findest du heraus mit:

    address

> Da Monero anonym ist, wirst du die Ursprungsadresse deiner empfangenen Gelder nicht sehen können. Wenn du sie aber doch aus irgendeinem Grund (beispielsweise, um einen besonderen Kunden zu honorieren) wissen möchtest, musst du dem Sender mitteilen, dass er eine Zahlungs-ID verwenden soll. Diese ist ein willkürlicher, optionaler Tag, mit welchem die Transaktion versehen wird. Um es dir einfach zu machen, kannst du eine Adresse erstellen, die bereits eine zufällige Zahlungs-ID enthält:

Reviewer: Not sure what "to credit a particular customer" means and if "einen besonderen Kunden honorieren" is the correct translation?

Author: I had my troubles with translating this very part. I couldn'd find a better German word than "honorieren" which could be used in this context (in my opinion, as I understood that you want to give back something to a good and well-paying customer, but I may be wrong of course). It may sound a bit stilted, so I'm up to ideas! :)

Author: @erciccione

Reviewer: As I understood it it is not about some "special, good and well-paying customer" but about all customers. Every customer's order gets its own unique payment id, otherwise the seller cannot relate payments to orders.

    integrated_address

Dies generiert eine zufällige Zahlungs-ID und gibt eine Adresse aus, welche dein eigenes Konto und zudem ebendiese Zahlungs-ID enthält. Wenn du eine bestimmte Zahlungs-ID verwenden möchtest, kannst du das folgendermaßen machen:

    integrated_address 12346780abcdef00

An eine von deinem Konto aus generierte integrierte Adresse gesendete Zahlungen gehen mit der Zahlungs-ID markiert auf dein Konto. Auf diese Weise kannst du Zahlungen unterscheiden.

## Getätigte Zahlungen gegenüber Dritten nachweisen

> Solltest du einen Händler bezahlt haben, dieser aber bestreiten, das Geld empfangen zu haben, kann es passieren, dass du deine Zahlung gegenüber Dritten nachweisen musst - etwa gegenüber ebenjenen Händlers, sofern es ein ehrlicher Fehler seinerseits war. Da Monero privat ist und man nicht ausmachen kann, wer Gelder empfangen oder gesendet hat, kannst du nicht einfach auf deine Transaktion in der Blockchain zeigen. Durch Bereitstellen des privaten, pro Transaktion vertraulichen Schlüssels ist es Dritten allerdings möglich, zu sehen, ob in dieser Transaktion Monero an die jeweilige Adresse gesendet wurden. Beachte, dass das Abspeichern dieser Transaktionsschlüssel standardmäßig deaktiviert ist. Wenn du denkst, du könntest diese Funktion gebrauchen, musst du sie vor dem Senden einschalten:

Reviewer: ```suggestion
Solltest du einen Händler bezahlt haben, dieser aber bestreiten, das Geld empfangen zu haben, kann es passieren, dass du deine Zahlung gegenüber Dritten nachweisen musst - oder sogar gegenüber des Händlers, sofern es ein ehrlicher Fehler seinerseits war. Da Monero privat ist und man nicht ausmachen kann, wer Gelder empfangen oder gesendet hat, kannst du nicht einfach auf deine Transaktion in der Blockchain verweisen. Durch Bereitstellen des privaten, pro Transaktion vertraulichen Schlüssels ist es Dritten allerdings möglich, zu sehen, ob in dieser Transaktion Monero an die jeweilige Adresse gesendet wurden. Beachte, dass das Abspeichern dieser Transaktionsschlüssel standardmäßig deaktiviert ist. Wenn du denkst, du könntest diese Funktion gebrauchen, musst du sie vor dem Senden einschalten:
```

    set store-tx-info 1

> Du kannst den Transaktionsschlüssel einer früheren Transaktion wiederherstellen:

Reviewer: ```suggestion
Du kannst den Transaktionsschlüssel einer früheren Transaktion abrufen:
```

Wenn du dieser Dritte bist (weil dir jemand beweisen möchte, dass er Monero an eine Adresse gesendet hat), kannst du den Nachweis folgendermaßen überprüfen:

    check_tx_key TXID TXKEY ADDRESS

Ersetze `TXID`, `TXKEY` und `ADDRESS` mit der dir bereitgestellten Transaktions-ID, dem jeweiligen Transaktionsschlüssel und der Zieladresse. monero-wallet-cli überprüft nun diese Transaktion und teilt dir mit, wie viele Monero in dieser Transaktion an die vorgegebene Adresse gesendet wurden.

## Möglichkeit des Bestätigens/Abbrechens von Zahlungen einrichten

Wenn du eine finale Bestätigungsmöglichkeit beim Senden einer Zahlung möchtest:

    set always-confirm-transfers 1

## Eine an dich gehende Zahlung finden

> Hast du eine Zahlung mit einer bestimmten Zahlungs-ID erhalten, kannst du diese folgendermaßen nachsehen:

Reviewer: ```suggestion
Hast du eine Zahlung mit einer bestimmten Zahlungs-ID erhalten, kannst du diese folgendermaßen aufsuchen:
```

    payments PAYMENTID

> Du kannst auch mehr als eine Zahlungs-ID vergeben.

Reviewer: ```suggestion
Du kannst auch mehr als eine Zahlungs-ID eingeben.
```
